app/models/member.rb

```diff
@@ -221,17 +221,10 @@ def self.nearest_to(place)
   end
 
   def update_newsletter_subscription
-    if confirmed_at_changed? && newsletter # just signed up
-      newsletter_subscribe
-    elsif confirmed_at # i.e. after member's confirmed their account
-      if newsletter_changed? # edited member settings
-        if newsletter
-          newsletter_subscribe
-        else
-          newsletter_unsubscribe
-        end
-      end
+    if newsletter
+      newsletter_subscribe confirmed_at_changed? || confirmed_at && newsletter_changed?
```

Comment:
Should there be an `if` between `newsletter_subscribe` and `confirmed_at_changed`? The first argument of `newsletter_subscribe` shouldn't be a boolean.

I'd also prefer brackets in the condition to make the precedence clear.